examples/tests/invalid/invalid_call_before_super.w

```diff
@@ -0,0 +1,44 @@
+class A {
+  a: num;
+  new(a: num) { this.a = a; }
+  get_a(): num { return this.a; }
+}
+
+class B extends A {
+  b:num;
+  new() {
+    this.b = 3;
+//  ^^^^^^^^^^^ This command cannot be executed before calling super()
+    super(5);
+  }
+}
+
+class C extends A {
+  new() {
+    super.get_a();
+//  ^^^^^^^^^^^^^^ This command cannot be executed before calling super()
+    super(5);   
+  }
+}
+
+class D extends A {
+    new(x: bool) {
+      if x {
+        super(1);
+//      ^^^^^^^^^ Call to super constructor can only be done from within class constructor
```

Comment:
This error message is a bit misleading because actually are within the class ctor, perhaps something like:
"Call to super constructor can only be done at the top level scope of the constructor"